plugins/outputs/cloudwatch/cloudwatch.go

```diff
@@ -30,9 +30,9 @@ import (
 )
 
 const (
-	defaultMaxDatumsPerCall        = 20     // PutMetricData only supports up to 20 data metrics per call by default
+	defaultMaxDatumsPerCall        = 1000   // PutMetricData only supports up to 1000 data metrics per call by default
 	defaultMaxValuesPerDatum       = 150    // By default only these number of values can be inserted into the value list
-	bottomLinePayloadSizeToPublish = 200000 // Leave 9600B for the last datum buffer. 200KB payload size, 5:1 compression ratio estimate.
+	bottomLinePayloadSizeToPublish = 500000 // Leave 2400B for the last datum buffer. 500KB payload size, 5:1 compression ratio estimate.
```

Comment:
Should sync up with internal: `bottomLinePayloadSizeToPublish = 999 000` (0.999 MB) and delete `5:1 compression ratio estimate`

Reply:
Can you confirm the API limit is actually 1 MiB, as in 1024*1024 Bytes?
If so then this limit could be raised to 1023 KiB (1023*1024 Bytes).
I don't think it is critical since a payload of 1,000 metric datums is unlikely to reach 1023 KiB.
May as well be as accurate as possible, right?